packages/compiler-cli/ngcc/src/main.ts

```diff
@@ -117,6 +117,17 @@ export interface SyncNgccOptions {
    * legacy message ids will all be stripped during translation.
    */
   enableI18nLegacyMessageIdFormat?: boolean;
+
+  /**
+   * Whether to read an entry-point manifest file from disk, rather than walking the directory tree
+   * looking for entry-points.
```

Comment:
This reads backwards to me 😇
How about inverting it:

```suggestion
   * Whether to walk the directory tree looking for entry-points rather than trying to read an
   * entry-point manifest file from disk.
```

Reply:
Yes, looking back this doesn't read well.

Reply:
Going for:

```
   * Whether to invalidate any entry-point manifest file that is on disk. Instead, walk the
   * directory tree looking for entry-points, and then write a new entry-point manifest, if
   * possible.
```